modules/caddytls/proxy.go

```diff
@@ -0,0 +1,3 @@
+package caddytls
+
+import _ "github.com/caddyserver/caddy/v2/modules/internal/network"
```

Comment:
Interesting; why the indirection here?

Reply:
This is embarrassing. I don't remember, and it still works without it.

Reply:
Done